staging/src/k8s.io/client-go/util/workqueue/doc.go

```diff
@@ -23,4 +23,4 @@ limitations under the License.
 //  * Multiple consumers and producers. In particular, it is allowed for an
 //      item to be reenqueued while it is being processed.
 //  * Shutdown notifications.
-package workqueue
+package workqueue // import "k8s.io/client-go/util/workqueue"
```

Comment:
@Toper2035 this is for client-go but the the title says that this for sample-controller.

https://github.com/kubernetes/kubernetes/pull/68783 updates the workqueue one. So can you fix the sample-controller ones in this? Thanks!

Reply:
@nikhita thanks, i have updated this PR's title.